src/app/views/App.spec.tsx

```diff
@@ -62,52 +64,193 @@ const renderApp = (args?: any) : any => {
       </IntlProvider>
     </Provider> );
 }
+window.fetch = jest.fn();
+window.open = jest.fn();
+
+
+// eslint-disable-next-line react/display-name
+jest.mock('../../app/views/query-runner/request/feedback/FeedbackForm.tsx', () => () => {
+  return <div />;
+});
 
 jest.mock('@microsoft/applicationinsights-react-js', () => ({
   // eslint-disable-next-line react/display-name
   withAITracking: () => React.Component,
   ReactPlugin: Object
-}))
+}));
 
-jest.mock('@ms-ofb/officebrowserfeedbacknpm/scripts/app/Window/Window', () => ({
-  OfficeBrowserFeedback: Object
-}))
+// eslint-disable-next-line no-console
+console.warn = jest.fn();
 
-jest.mock('@ms-ofb/officebrowserfeedbacknpm/Floodgate', () => ({
-  makeFloodgate: Object
-}))
+describe('It should render the main GE site', () => {
```

Comment:
There are conventions for naming tests so that they are easy to understand. Like:` It/Should` or `Given/When/Then`

In this case it looks like we are using It/Should

So that the tests are easy to understand and also to make them read like logical prose, we should look at `Describe` as the title/ first part of the sentence of the group of tests

Like `​Describe('App rendering')​`

then `​it` sections​ should talk more about what was described like: `​it('should return x when y')​`

Reply:
I noticed that most of the tests did not follow this convention. I have fixed them.
It's good you raised it!